src/vmm/src/devices/virtio/queue.rs

```diff
@@ -34,6 +34,8 @@ pub enum QueueError {
     DescIndexOutOfBounds(u16),
     /// Failed to write value into the virtio queue used ring: {0}
     MemoryError(#[from] vm_memory::GuestMemoryError),
+    /// Pointer is not aligned properly: {0:p} not {1}-byte aligned.
+    PointerNotAligned(*const u8, u8),
```

Comment:
Rust really does not like passing pointers, so let's just convert them to `usize` and print them in hex `{:#x}`.